apps/web/app/(app)/gardens/[chain]/[garden]/[community]/[poolId]/[proposalId]/page.tsx

```diff
@@ -297,25 +297,6 @@ export default function Page({
             />
           </>
         }
-        <div className="absolute top-8 right-10">
```

Comment:
Wuutt ?

Reply:
not sure if its what we want right ?

Reply:
is was doble coded because of a merge error goss!